ethon.gemspec

```diff
@@ -20,7 +20,10 @@ Gem::Specification.new do |s|
 
   s.add_dependency('ffi', ['>= 1.15.0'])
 
-  s.files        = `git ls-files`.split("\n")
-  s.test_files   = `git ls-files -- spec/*`.split("\n")
+  s.files = Dir.chdir(__dir__) do
+    `git ls-files -z`.split("\x0").reject do |file|
+      file.start_with?(*%w[. Gemfile Guardfile Rakefile ethon.gemspec profile spec])
```

Comment:
The `ethon.gemspec` should remain. Please drop that from the list and we're ready to merge.

Reply:
Updated in [0f699a2](https://github.com/typhoeus/ethon/pull/227/commits/0f699a20b31bc9b1567426e57786f3fb8b4fe524).